src/services.js

```diff
@@ -186,6 +186,16 @@ export class Services {
   }
 
   /**
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.